test-utils/state-viewer/Cargo.toml

```diff
@@ -9,6 +9,7 @@ edition = "2021"
 ansi_term = "0.12"
 borsh = "0.9"
 clap = "=3.0.0-beta.2"
+itertools = "0.10.0"
```

Comment:
I agree with @matklad it's good to try to minimize number of imports. Especially, if they are just used once, and can be replaced with a slightly different syntax.